templates/default/ndb-agent/config.yml.erb

```
@@ -4,6 +4,9 @@ general:
   cluster-topology-location: <%= node['cloud']['ndb-agent']['config'] %>/cluster-topology.json
   backup-staging-directory: <%= node['ndb']['BackupDataDir'] %>
   extra-mysqld-vms: 1
+  public-state-filepath: /srv/hops/ndb-agent/ndb-agent/config/public-state.json
```

`/srv/hops/ndb-agent/ndb-agent/config` should not be hardcoded

```suggestion
  public-state-filepath: <%= node['cloud']['ndb-agent']['config'] %>/public-state.json
```